machine-types_5b24032b0428632c466af399.md

```diff
@@ -1,16 +1,13 @@
-- [Overview](#overview)
-- [Linux machine types](#linux-machine-types)
-- [Apple machine type](#apple-machine-type)
-
-## Overview
-
 A **machine type** specifies a particular collection of virtualized
 hardware resources available to a virtual machine (VM) instance,
 including the memory size, virtual CPU count, and disk.
 
 This guide describes the available machine types. For using the supported
```

Comment:
Let's also fix this sentence: _For using..._ -> _For instructions about using..._